task/tasks.json

```diff
@@ -157,6 +157,17 @@
       "Storage"
     ]
   },
+  "MakeBucket": {
+    "type": "dependent",
+    "path": "bucket_make_bucket",
```

Comment:
Let's use the `<verb>[-<noun>]` pattern for files.

`make_bucket` looks good to me.